### PR TITLE
Clean up repo

### PR DESCRIPTION
**Pull request by radadjoneva**
Removed unused files (stored in extended_version branch) and added some samples files, which are excluded from the processing.

### CODE REVIEW

**Review by avanijamenon — APPROVED** [review]

File: rna/median_expression_files/.gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Ignore everything in this directory
 *
 # But not this file!
-!.gitignore
\ No newline at end of file
+!.gitignore
+!.sample_rna_expression_homo_sapiens.csv
```
Comment (HelenaFP) [inline rna/median_expression_files/.gitignore:5]: same comment here

File: rna/quant_files/processed/.gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Ignore everything in this directory
 *
 # But not this file!
-!.gitignore
\ No newline at end of file
+!.gitignore
+!.sample_homo_sapiens.csv
```
Comment (HelenaFP) [inline rna/quant_files/processed/.gitignore:5]: same comment here

**Review by HelenaFP — COMMENTED** [review]
I would recommend 1 single .gitigonore with all the information.
Is that possible?
If not, what is the reasoning behind having several of these files?

File: dna/csv_files/.gitignore [hunk 3]
```diff
@@ -2,3 +2,4 @@
 *
 # But not this file!
 !.gitignore
+!.sample_data_homo_sapiens.csv
```
[the remark this answers is not in the document]
Comment (radadjoneva) [reply dna/csv_files/.gitignore:5]: The reason why I had made this was to make sure the folder is present on the remote repo, but now that we have a sample file we can remove the .gitignore

**Pull request merged**